src/applications/facility-locator/containers/UrgentCareAlert.jsx

```diff
@@ -1,34 +0,0 @@
-import React from 'react';
```

Comment:
This alert is not used; Slack thread here: https://dsva.slack.com/archives/C0FQSS30V/p1723150747496079